src/documentation/pages/components/transfer-list/examples/_base.vue

```diff
@@ -0,0 +1,180 @@
+<template lang="pug">
+ComponentExample(title="Base" id="base" padding="-p--0" :tabs="exampleTabs")
+  template(#example)
+    ChiTransferList(:transferListData='transferListData', :config='config')
+  template(#code-vue)
+    Copy(lang="html" :code="codeSnippets.vue")
+</template>
+
+<script lang="ts">
+import { Vue } from 'vue-facing-decorator';
+
+@NuxtComponent({
+  data: () => {
+    return {
+      exampleTabs: [
```

Comment:
Please move all the props to the class definition, if in the future we need to add a prop in the class, we will have to move all the props defined in the decorator.

Reply:
Done